src/NLU.DevOps.Core.Tests/LabeledUtteranceConverterTests.cs

```diff
@@ -168,7 +191,7 @@ public static void ThrowsWithInvalidEndPos()
             toObject.Should().Throw<InvalidOperationException>();
 
             entityJson["endPos"] = 4;
-            toObject.Should().Throw<InvalidOperationException>();
+            toObject.Should().NotThrow<InvalidOperationException>();
```

Comment:
Can you put this in it's own test case called "DeserializesSingleCharacterEntities" or something?

Reply:
See comment above, let's re-use this code to test the less than case, so set L193 to `3`.